go/tools/wtool/main.go

```diff
@@ -0,0 +1,144 @@
+/*wtool augments your bazel WORKSPACE file with new_go_repository entries
+
+Example Usage: wtool com_github_golang_glog com_google_cloud_go
+will add 2 new_go_repository to your WORKSPACE
+by converting com_github_golang_glog -> github.com/golang/glog
+and so forth and then doing a 'git ls-remote' to get
+the latest commit.
+
+if wtool cannot figure out the bazel -> Go mapping, try
+Other Usage: wtool -asis github.com/golang/glog
```

Comment:
Perhaps we should rewrite the docs to just list every flag (e.g. https://godoc.org/golang.org/x/tools/cmd/godoc). This will make it easier to add docs for new flags later.

However since there is only 1 flag at the moment, feel free to punt on this.